keyboards/tzarc/djinn/graphics/theme_djinn_default.inl.c

```diff
@@ -0,0 +1,237 @@
+// Copyright 2018-2022 Nick Brassel (@tzarc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "djinn.qgf.c"
+#include "lock-caps-ON.qgf.c"
+#include "lock-scrl-ON.qgf.c"
+#include "lock-num-ON.qgf.c"
+#include "lock-caps-OFF.qgf.c"
+#include "lock-scrl-OFF.qgf.c"
+#include "lock-num-OFF.qgf.c"
+#include "thintel15.qff.c"
```

Comment:
Still not a fan of this.  In generally, we've actively discouraged including C files, in part because it's a bad practice.   And ... it causes compile issues, if they're changed, as it may not be recompiled properly.